corehq/util/doc_processor/sql.py

```diff
@@ -48,14 +45,10 @@ def resumable_sql_model_iterator(iteration_key, reindex_accessor, chunk_size=100
     is stopped and later resumed.
     """
 
-    def data_function(from_db, filter_value, last_id):
-        return reindex_accessor.get_docs(from_db, filter_value, last_id, limit=chunk_size)
+    def data_function(from_db, last_id):
```

Comment:
Changing this function signature made it not possible to resume iterations that were in progress (ongoing blob db migration) because [`ResumableArgsProvider`](https://github.com/dimagi/commcare-hq/blob/6143fb5b3e09a969b1a50e932550bf2360a2f27b/corehq/util/pagination.py#L131-L132) uses persisted iteration state (args passed to this function) from the old code to resume iteration.

Reply:
Apologies for that, I completely forgot about those migrations. we can revert this if necessary